agreements/sponsorship/README.md

```diff
@@ -18,26 +18,29 @@ Each annual sponsorship provides these benefits:
 
 - Logo recognition on our website and promotional materials
 - Public acknowledgment through our social media channels
+- If sponsorship is designated to support a specific Commonhaus Project, your logo may also appear on that Project’s website
```

Comment:
Just to clarify "support" here means provide some form of commercial support (consulting, patching, releases etc), right ?

Not support in general sense ? Should that be more explicit then?

And side question - and i might just be missing context. Wouldn't commercial support or any one actually be allowed to be on that project website independent of sponsoring via commonhaus ? Or do we now make this a requirement to be listed on project websites?

Reply:
Yes. Some form of commercial support.

The latter. RHBQ, e.g., would not be listed on the Quarkus website unless RH sponsors the foundation (and/or the project as part of the foundation). 

FWIW, this idea is from Sanne.